### PR TITLE
feat: devcontainer proposal

### PR DESCRIPTION
**Pull request by VictorAvelar**
## Description

Provide a devcontainer for VSCode integration.

## Motivation and context

Ease the development experience, which is especially appealing for users with limited local settings who want to avoid installing golang natively.

## How has this been tested?

This PR was created using the devcontainer.

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

Go over all the following points, and put an `x` in all the boxes that apply.

Please, please, please, don't send your pull request until all of the boxes are ticked. Once your pull request is created, it will trigger a build on our continuous integration server to make sure your [tests and code style pass](https://help.github.com/articles/about-required-status-checks/).

- [x] I have read the **[CONTRIBUTING](CONTRIBUTING.md)** document.
- [x] My pull request addresses exactly one patch/feature.
- [x] I have created a branch for this patch/feature.
- [x] Each individual commit in the pull request is meaningful.
- [x] I have added tests to cover my changes.
- [x] If my change requires a change to the documentation, I have updated it accordingly.

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        }
+    },
+    "portsAttributes": {
+        "9000": {
+            "label": "Hello Remote World",
+            "onAutoForward": "notify"
+        }
+    },
+    "postCreateCommand": "go mod download"
+}
```
Comment (Bird87ZA) [inline .devcontainer/devcontainer.json:74]: new line

**Review by Bird87ZA — CHANGES_REQUESTED** [review]

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (VictorAvelar) [reply .devcontainer/devcontainer.json:74]: This file is JSON 🤔

Comment (Bird87ZA) [reply .devcontainer/devcontainer.json:74]: This doens't change the facts. It's always [recommended](https://unix.stackexchange.com/a/18789/217743) [to have](https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file) [a new line](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) at the end of a file.

**Comment by Bird87ZA** [comment]
<3

**Pull request merged**